project/cmake/modules/FindLCMS2.cmake

```diff
@@ -0,0 +1,47 @@
+#.rst:
+# FindLCMS2
+# -----------
+# Finds the LCMS Color Management library
+#
+# This will will define the following variables::
+#
+# LCMS2_FOUND - system has LCMS Color Management
+# LCMS2_INCLUDE_DIRS - the LCMS Color Management include directory
+# LCMS2_LIBRARIES - the LCMS Color Management libraries
+# LCMS2_DEFINITIONS - the LCMS Color Management definitions
+#
+# and the following imported targets::
+#
+#   LCMS2::LCMS2   - The LCMS Color Management library
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_LCMS2 lcms2 QUIET)
+endif()
+
+find_path(LCMS2_INCLUDE_DIR NAMES lcms2.h
+                            PATHS ${PC_LCMS2_INCLUDEDIR})
+find_library(LCMS2_LIBRARY NAMES lcms2 liblcms2
+                           PATHS ${PC_LCMS2_LIBDIR})
+
+set(LCMS2_VERSION ${PC_LCMS2_VERSION})
```

Comment:
As long as we don't request specific versions, it doesn't cause any harm. The advantage is that fphsa below will show the version number if it's set. (and just omits it if it wasn't found through pkgconfig)